index.bs

```diff
@@ -77,16 +77,18 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The magnetometer's associated {{Sensor}} subclass is the {{Magnetometer}} class.
+The magnetometer's associated {{Sensor}} subclass is the {{Magnetometer}} class or the {{RawMagnetometer}} class,
```

Comment:
That sounds weird.

There are two magnetomer sensors which each are defined by their subclasses, either Magnetomer of RawMegnetometer, depending on whether they expose calibrate...